Gruntfile.js

```diff
@@ -14,7 +14,7 @@ module.exports = function(grunt) {
                     script: "index.js",
                     timeout: 3000,
                     harmony: true,
-                    args: ['-s'] //to more traces set -vvv instead of -s (silent)
+                    args: ['-vvv'] //to more traces set -vvv instead of -s (silent)
```

Comment:
I would not like to get anything to console except test framework prints -> if this still affect to that please revert back to `-s` .